sources/osgViewer/input/source/InputSourceKeyboard.js

```diff
@@ -43,6 +43,14 @@ utils.createPrototypeObject(
             if (!parsedEvent.action) {
                 return true;
             }
+            if (!nativeEvent.key) {
+                // Sentry reports errors with the nativeEvent key undefined.
+                // We can't reproduce the issue so this code is supposed to give us more information when that happens.
+                var error = new Error('Invalid keyboard event', 'InputSourceKeyboard.js', 47);
```

Comment:
I would add a JSON.stringify(nativeEvent) concatened with the "inputsource.js"

Reply:
```var error = new Error('Invalid keyboard event', 'InputSourceKeyboard.js: ' + JSON.stringify(nativeEvent), 47);```

Reply:
I added it on the cause attribute thinking we could see it on sentry.
But if you prefer like that I'm ok

Reply:
sentry gets only strings so you need to stringify the object NativeEvent to be able to "debug" from sentry. (you don't get "variable stack", only "function stack" traces in sentry)